--- sha1.c.orig	2025-03-27 21:29:33
+++ sha1.c	2025-03-27 21:29:51
@@ -27,14 +27,9 @@
 
 #include <stddef.h>
 #include <string.h>
-#include <endian.h>
 
-#if __BYTE_ORDER == __BIG_ENDIAN
-# define SWAP(n) (n)
-#else
 # define SWAP(n) \
     (((n) << 24) | (((n) & 0xff00) << 8) | (((n) >> 8) & 0xff00) | ((n) >> 24))
-#endif
 
 /* This array contains the bytes used to pad the buffer to the next
    64-byte boundary.  (RFC 1321, 3.1: Step 1)  */
